sdk/schemaregistry/Azure.Data.SchemaRegistry/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0100:ConfigureAwait(false) must be used.", Justification = "<Pending>", Scope = "member", Target = "~M:Azure.Data.SchemaRegistry.SchemaRestClient.GetByIdAsync(System.String,System.Threading.CancellationToken)~System.Threading.Tasks.Task{Azure.Core.ResponseWithHeaders{System.String,Azure.Data.SchemaRegistry.SchemaGetByIdHeaders}}")]
```

Comment:
This was a workaround for this: https://github.com/Azure/autorest.csharp/pull/939

Once the generator is updated in this repo, this file can be deleted.